src/main/scala/org/mauritania/main4ino/cli/Algebras.scala

```diff
@@ -14,13 +18,11 @@ object Algebras {
 
   trait Configs[F[_]]  {
 
-    def user(c: Config, u: AddRawUser): User = {
-      val hashed = Auther.hashPassword(u.pass, c.salt)
-      User(u.name, hashed, u.email, u.granted)
-    }
+    def user(c: Config, u: AddRawUser)(implicit P: PasswordHasher[F, BCrypt], M: Monad[F]): F[User] = for {
```

Comment:
Given a `Config` and a `AddRawUser` command, I'd expect the method `user(...)` to be referentially transparent, and generate a well known `User`, however you use `F[User]` as return type instead of just `User`. Isn't the method RT?